src/main/src/main/java/org/geoserver/threadlocals/EnvVarThreadLocalTransfer.java

```diff
@@ -0,0 +1,30 @@
+/* (c) 2018Open Source Geospatial Foundation - all rights reserved
```

Comment:
Typo here (not a deadly problem :) ).
I would add a whitespace between year and Organization